commands/team_e2e_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestDeleteTeamsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Error deleting team which does not exist", func(c client.Client) {
+		printer.Clean()
+		teamName := "existingName"
+		cmd := &cobra.Command{}
+		args := []string{""}
+		args[0] = teamName
```

Comment:
We can initialize and assign in the same line here (and in the following tests too):

```
args := []string{teamName}
```

Reply:
Done.